pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile

```diff
@@ -61,16 +61,21 @@ ansiColor('xterm') {
 
     stage('[Job] Provision environment') {
       script {
-        build job: createNamespaceJobName,
-          parameters: _defaultEnvironmentParameters(namespace.getClusterName(), namespace.getNamespaceName()) +
-            [string(name: 'FOLIO_BRANCH', value: folioBranch),
-             string(name: 'OKAPI_VERSION', value: okapiVersion),
-             string(name: 'CONFIG_TYPE', value: 'testing'),
-             booleanParam(name: 'LOAD_REFERENCE', value: true),
-             booleanParam(name: 'LOAD_SAMPLE', value: true),
-             booleanParam(name: 'CONSORTIA', value: true),
-             booleanParam(name: 'GREENMAIL', value: false),
-             string(name: 'MEMBERS', value: '')]
+        try {
+          build job: createNamespaceJobName,
+            parameters: _defaultEnvironmentParameters(namespace.getClusterName(), namespace.getNamespaceName()) +
+              [string(name: 'FOLIO_BRANCH', value: folioBranch),
+               string(name: 'OKAPI_VERSION', value: okapiVersion),
+               string(name: 'CONFIG_TYPE', value: 'testing'),
+               booleanParam(name: 'LOAD_REFERENCE', value: true),
+               booleanParam(name: 'LOAD_SAMPLE', value: true),
+               booleanParam(name: 'CONSORTIA', value: true),
+               booleanParam(name: 'GREENMAIL', value: false),
+               string(name: 'MEMBERS', value: '')]
+        } catch (Exception new_ex) {
+          println('Failed to provision environment: ' + new_ex)
+          slackNotifications.sendPipelineFailSlackNotification("#rancher_tests_notifications")
+        }
```

Comment:
Here you need to propagate the error as after slack notification, the send pipeline will continue execution of the following stages.

Reply:
I've added the "throw new Exception"